### ISSUE TITLE
iam access-group-policy-create inline help is badly rendered if there are any args

### ISSUE THREAD

**Issue by brackendawson** [IBM-Cloud/ibm-cloud-cli-release#206]
The `ibmcloud iam access-group-policy-create` command has this for its inline help:
```
% ibmcloud iam access-group-policy-create SQUAD -h
${COMMAND_FULL_NAME} GROUP_NAME {--file @JSON_FILE | --roles ROLE_NAME1,ROLE_NAME2... [--service-name SERVICE_NAME] [--service-instance SERVICE_INSTANCE_GUID] [--region REGION] [--resource-type RESOURCE_TYPE] [--resource RESOURCE] [--resource-group-name RESOURCE_GROUP_NAME] [--resource-group-id RESOURCE_GROUP_ID] [--tags name1:value1,name2:value2...] [--account-management] [--attributes name=value,name=value...]}} [--output FORMAT] [-q, --quiet] [--api-version v1 | v2]

EXAMPLE:
    ${COMMAND_FULL_NAME} example_group --file policy.json
        Create an access group policy from a JSON file
    ${COMMAND_FULL_NAME} example_group --file policy.json --api-version v2
        Create an access group policy from a JSON file using Access Policy API Version 2
    ${COMMAND_FULL_NAME} example_group --roles Administrator --service-name sample-service
        Give example_group Administrator role for all sample-service resources
    ${COMMAND_FULL_NAME} example_group --roles Administrator --service-name sample-service --api-version v2
        Give example_group Administrator role for all sample-service resources using Access Policy API Version 2
    ${COMMAND_FULL_NAME} example_group --roles Editor --service-name sample-service --service-instance d161aeea-fd02-40f8-a487-df1998bd69a9 --region us-south --resource-type key --resource key123
        Give example_group Editor role for resource key123 of sample service instance with GUID d161aeea-fd02-40f8-a487-df1998bd69a9 in us-south region
    ${COMMAND_FULL_NAME} example_group --roles Operator --resource-type resource-group --resource dda27e49d2a1efca58083a01dfde18f6
        Give example_group Operator role for resource group with ID dda27e49d2a1efca58083a01dfde18f6
    ${COMMAND_FULL_NAME} example_group --roles Viewer --resource-group-name sample-resource-group
        Give example_group Viewer role for the members of resource group sample-resource-group
    ${COMMAND_FULL_NAME} example_group --roles Viewer --resource-group-id dda27e49d2a1efca58083a01dfde18f6
        Give example_group Viewer role for the members of resource group with ID dda27e49d2a1efca58083a01dfde18f6
    ${COMMAND_FULL_NAME} example_group --roles Administrator --account-management
        Give example_group Administrator role for all account management services
    ${COMMAND_FULL_NAME} example_group --roles Viewer
        Give example_group Viewer role for all resources in account
    ${COMMAND_FULL_NAME} example_group --roles Viewer --service-name is --attributes "instanceId=*"
        Give example_group Viewer role for service "is" resources with attribute "instanceId" equal to "*"
        To use advanced operators to create wildcard policies, use a JSON policy document with the "--file" option. For more information about creating wildcard policies, see https://cloud.ibm.com/docs/account?topic=account-wildcard#policy-js

Usage:
  ibmcloud iam access-group-policy-create [flags]

Flags:
      --account-management           Give access to all account management services
      --api-version string           Version of the access policy API (Default: v1) (default "v1")
      --attributes stringToString    Set resource attributes in the form of 'name=value,name=value....' (default [])
      --file string                  JSON file of policy definition
  -h, --help                         help for access-group-policy-create
      --output OutputFormat          Specify output format, only 'JSON' is supported.
      --region string                Region of the policy definition. This option is exclusive with '--file'. For supported regions, run '${COMMAND_NAME} regions'.
      --resource string              Resource of the policy definition. This option is exclusive with '--file'.
      --resource-group-id string     ID of the resource group. '*' means all resource groups. This option is exclusive with '--file' and '--resource-group-name'.
      --resource-group-name string   Name of the resource group. '*' means all resource groups. This option is exclusive with '--file' and '--resource-group-id'.
      --resource-type string         Resource type of the policy definition. This option is exclusive with '--file'.
  -r, --roles string                 Role names of the policy definition. For supported roles of a specific service, run '${COMMAND_NAME} iam roles --service SERVICE_NAME'. This option is exclusive with '--file'.
      --service-instance string      GUID of service instance of the policy definition. This option is exclusive with '--file'.
      --service-name string          Service name of the policy definition. This option is exclusive with '--file'.
      --tags string                  Access tags of the resource.

```

Note the instances of `${COMMAND_FULL_NAME}` in the output.

If you run it with `--help` as the only argument then it works, but with any number of arguments and `--help` then it does the above.

### Env
```
% ibmcloud version                                                                                                                                                                                                             14:38:27
ibmcloud version 2.24.0+b6c228b-2024-03-21T18:54:40+00:00
alandaws@brackens-mbp ~/src/github.ibm.com/alchemy-registry/registry-charts (master) <healthcheck-dev> {registry-dev}
% ibmcloud plugin list                                                                                                                                                                                                         16:24:34
Listing installed plug-ins...

Plugin Name                                     Version          Status             Private endpoints supported
cloudant[cl]                                    0.2.0                               true
code-engine[ce]                                 1.49.5           Update Available   true
container-registry[cr]                          1.3.5            Update Available   true
container-service[kubernetes-service/ks]        1.0.601                             false
event-streams[es]                               2.7.2312131114                      true
sl                                              1.5.1            Update Available   false
vpc-infrastructure[infrastructure-service/is]   9.0.3            Update Available   true

alandaws@brackens-mbp ~/src/github.ibm.com/alchemy-registry/registry-charts (master) <healthcheck-dev> {registry-dev}
% uname -a                                                                                                                                                                                                                     16:24:45
Darwin brackens-mbp.hursley.uk.ibm.com 23.4.0 Darwin Kernel Version 23.4.0: Fri Mar 15 00:10:42 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6000 arm64
```

**Comment by steveclay** [comment]
Thanks for pointing this out. We will investigate.

**Comment by steveclay** [comment]
This is addressed as of CLI release 2.28.0.
